analyses/chromosomal-instability/util/circos-plots.R

```diff
@@ -135,6 +135,13 @@ circos_map_plot <- function(df,
   y_min <- min(bed_df$y_val, na.rm = TRUE)
   y_max <- max(bed_df$y_val, na.rm = TRUE)
 
+  # Can't have identical y_min and y_max, this is just so CircleCI runs even if 
+  # the subset data is wonky
+  if (y_min == y_max) {
+    y_max <- y_max + 0.001 
```

Comment:
y_min and y_max are calculated based on the data, but with some of the subsets, you can get values that are identical and circlize doesn't like that. So this is my fix.